DependencyInjection/Configuration.php

```diff
@@ -65,9 +69,10 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('class')
+                    ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('group')->cannotBeEmpty()->end()
-                        ->scalarNode('user')->cannotBeEmpty()->end()
+                        ->scalarNode('group')->cannotBeEmpty()->defaultValue(BaseGroup::class)->end()
```

Comment:
This will work with the default entities?

Reply:
> This will work with the default entities?

I see no reason why it wouldn't work.

Reply:
You're right. Thought this was an abstract class